resolvers/ambulance.js

```diff
@@ -1,34 +1,48 @@
 'use strict';
 
 const db = require('../models');
+const { Roles } = require('../utils/enum');
 const validators = require('../utils/validators');
 
 const ambulanceResolvers = {
   Query: {
-    ambulances: () =>
-      db.ambulance.findAll({
+    ambulances: () => {
+      validators.validateRole(
+        [Roles.COMMANDER, Roles.SUPERVISOR, Roles.DISPATCH],
```

Comment:
Small nitpick - maybe change this to `Object.keys(Roles)` or `Object.values(Roles)`, so we wouldn't have to add more items to this array if we were to add more Roles in the future. 

This might be intentional though - maybe we only want these three roles. Just something to think about

Reply:
this is a good point - for these ones where we essentially just validate that they have a role, doing `Object.values(Roles)` makes a lot of sense

Reply:
Sounds good!